test/dataset/test_unsupervised_dataset.py

```diff
@@ -47,3 +48,14 @@ def test_on_the_fly_feature_extraction_unsupervised_dataset(libri_cut_set):
     # E              [-13.2, -11.2,  -9.6, ...,  -5.6,  -6.5,  -7.6],
     # E              [-12. , -10.1, -10.1, ...,  -5.8,  -7. ,  -7.8],...
     np.testing.assert_array_almost_equal(ref_feats, tested_feats, decimal=0)
```

Comment:
again, not so sure about this decimal=0 thing.

Reply:
In case somebody stumbles upon this in the future - we discussed and analyzed this in #55